app/models/topic.rb

```diff
@@ -13,6 +13,10 @@ class Topic < ActiveRecord::Base
 
   default_scope { includes(:translations) }
 
+  scope :with_galleries, -> do
+    includes(:translations).joins(:categorisations).joins('INNER JOIN galleries on (categorisations.categorisable_id = galleries.id)')
```

Comment:
1. You can omit `includes(:translations)` here as it's inherited from the default scope.
2. You also need a join condition of `categorisations.categorisable_type="Gallery"`